### PR TITLE
HTML-mail.

### PR DESCRIPTION
**Pull request by GheorghitaVlad**

### CODE REVIEW

File: _html/templates/mail.html [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+                .templateContainer {
+                    width: 100% !important;
+                }
+                
+                img {
+                    max-width: 100% !important;
+                }
+
+                .block {
+                    display: inline-block!important;
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add space before !important.

File: _html/templates/mail.html [hunk 2]
```diff
@@ -0,0 +1,574 @@
...
+                    width: 100%!important;
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add space before !important.

File: _html/templates/mail.html [hunk 3]
```diff
@@ -0,0 +1,574 @@
...
+                    text-align: center!important;
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add space before !important.

File: _html/templates/mail.html [hunk 4]
```diff
@@ -0,0 +1,574 @@
...
+                }
+
+                .mobile-reposition {
+                    text-align: center!important;
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add space before !important.

File: _html/templates/mail.html [hunk 5]
```diff
@@ -0,0 +1,574 @@
...
+            }
+
+        </style>
+    </head>
+    <body bgcolor="#ffffff" style="text-align: center; background-color: #ffffff;">
+        <center>
+            <table border="0" cellpadding="0" cellspacing="0" width="100%" bgcolor="#ffffff" class="content-wrapper">
+                <tr style="text-align: center;">
+                    <td>
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="600px" class="templateContainer" bgcolor="#200F17">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 6]
```diff
@@ -0,0 +1,574 @@
...
+                                            </td>
+                                        </tr>
+                                    </table>
+                                    <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                        <tr>
+                                            <td style="padding: 15px;">
+                                                <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                                    <tr>
+                                                        <td>
+                                                            <span></span>
```
Comment (bajenaru) [inline _html/templates/mail.html]: Is this needed? If not please remove it, otherwise ignore the comment.

File: _html/templates/mail.html [hunk 7]
```diff
@@ -0,0 +1,574 @@
...
+                                                            </table>
+                                                        </td>
+                                                    </tr>
+                                                </table>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                <table border="0" cellpadding="0" cellspacing="0" width="100%" class="buttonContainer" style="min-width:100%;">
+                                                    <tbody class="buttonOuter">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 8]
```diff
@@ -0,0 +1,574 @@
...
+                                                        <tr>
+                                                            <td style="padding-top:0; padding-right:18px; padding-bottom:18px; padding-left:18px;" valign="top" align="center" class="mcnButtonBlockInner">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 9]
```diff
@@ -0,0 +1,574 @@
...
+                                                                <table border="0" cellpadding="0" cellspacing="0" class="mcnButtonContentContainer" style="border-collapse: separate !important;border-radius: 3px;background-color: #FCB02A;">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 10]
```diff
@@ -0,0 +1,574 @@
...
+                                                                    <tbody>
+                                                                        <tr>
+                                                                            <td align="center" valign="middle" class="mcnButtonContent" style="font-family: Helvetica; font-size: 18px; padding: 18px;">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html (hunk 1 above)
Comment (GheorghitaVlad) [reply _html/templates/mail.html]: I use them as "spaces" between content.

File: _html/templates/mail.html [hunk 11]
```diff
@@ -0,0 +1,574 @@
...
+                                                                                <a class="Button" href="http://" target="_self" style="font-weight: bold;letter-spacing: -0.5px;line-height: 100%;text-align: center;text-decoration: none;color: #FFFFFF;">Verify email address</a>
```
Comment (bajenaru) [inline _html/templates/mail.html]: Use lowercase naming for class .

File: _html/templates/mail.html (hunk 6 above)
Comment (GheorghitaVlad) [reply _html/templates/mail.html]: i use them as "spaces " between elements

File: _html/templates/mail.html [hunk 12]
```diff
@@ -0,0 +1,574 @@
...
+                                                                </tr>
+                                                            </table>
+                                                        </td>
+                                                </table>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                                                    <tbody class="buttonOuter">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 13]
```diff
@@ -0,0 +1,574 @@
...
+                                                        </td>
+                                                    </tr>
+                                                </table>
+                                                <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                                    <tr>
+                                                        <td class="block" style="width: 45%;">
+                                                            <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                                                <tr style="text-align: right;" class="mobile-reposition">
+                                                                    <td>
+                                                                        <img src="http://frontend-news.dev.pitechplus.com/drupal_camp_mail/transylvania.png">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add alt attribute.

File: _html/templates/mail.html [hunk 14]
```diff
@@ -0,0 +1,418 @@
...
+                                                                    <td>
+                                                                        <table>
+                                                                            <tr>
+                                                                                <td>
+                                                                                    <table>
+                                                                                        <tr>
+                                                                                            <td>
+                                                                                                <table align="left" border="0" cellpadding="0" cellspacing="0" style="display:inline;">
+                                                                                                    <tbody><tr>
+                                                                                                        <td valign="top" style="padding-right:10px; padding-bottom:9px;" class="mcnFollowContentItemContainer">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 15]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnFollowContentItem">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 16]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                <tbody><tr>
+                                                                                                                    <td align="left" valign="middle" style="padding-top:5px; padding-right:10px; padding-bottom:5px; padding-left:9px;">
+                                                                                                                        <table align="left" border="0" cellpadding="0" cellspacing="0" width="">
+                                                                                                                            <tbody>
+                                                                                                                                <tr>                                                        
+                                                                                                                                    <td align="center" valign="middle" width="24" class="mcnFollowIconContent">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 17]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                                        <a href="http://www.twitter.com/" target="_blank">
+                                                                                                                                            <img src="http://frontend-news.dev.pitechplus.com/drupal_camp_mail/facebook.png" style="display:block;" height="24" width="24" class="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add alt attribute and remove class attribute if empty.

File: _html/templates/mail.html [hunk 18]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            </td>
+                                                                                                        </tr>
+                                                                                                    </tbody>
+                                                                                                </table>
+                                                                                            </td>
+                                                                                            <td>
+                                                                                                <table align="left" border="0" cellpadding="0" cellspacing="0" style="display:inline;">
+                                                                                                    <tbody><tr>
+                                                                                                        <td valign="top" style="padding-right:10px; padding-bottom:9px;" class="mcnFollowContentItemContainer">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 19]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnFollowContentItem">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 20]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                <tbody><tr>
+                                                                                                                    <td align="left" valign="middle" style="padding-top:5px; padding-right:10px; padding-bottom:5px; padding-left:9px;">
+                                                                                                                        <table align="left" border="0" cellpadding="0" cellspacing="0" width="">
+                                                                                                                            <tbody>
+                                                                                                                                <tr>                                                        
+                                                                                                                                    <td align="center" valign="middle" width="24" class="mcnFollowIconContent">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 21]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                                        <a href="http://www.twitter.com/" target="_blank">
+                                                                                                                                            <img src="http://frontend-news.dev.pitechplus.com/drupal_camp_mail/google.png" style="display:block;" height="24" width="24" class="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Add alt attribute and remove class attribute if empty.

File: _html/templates/mail.html [hunk 22]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            </td>
+                                                                                                        </tr>
+                                                                                                    </tbody>
+                                                                                                </table>
+                                                                                            </td>
+                                                                                            <td>
+                                                                                                <table align="left" border="0" cellpadding="0" cellspacing="0" style="display:inline;">
+                                                                                                    <tbody><tr>
+                                                                                                        <td valign="top" style="padding-right:10px; padding-bottom:9px;" class="mcnFollowContentItemContainer">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 23]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnFollowContentItem">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 24]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                <tbody><tr>
+                                                                                                                    <td align="left" valign="middle" style="padding-top:5px; padding-right:10px; padding-bottom:5px; padding-left:9px;">
+                                                                                                                        <table align="left" border="0" cellpadding="0" cellspacing="0" width="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Remove empty width property.

File: _html/templates/mail.html [hunk 25]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                            <tbody>
+                                                                                                                                <tr>                                                        
+                                                                                                                                    <td align="center" valign="middle" width="24" class="mcnFollowIconContent">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 26]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                                        <a href="http://www.twitter.com/" target="_blank">
+                                                                                                                                            <img src="http://frontend-news.dev.pitechplus.com/drupal_camp_mail/twitter.png" style="display:block;" height="24" width="24" class="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Remove class and add alt attribute.

File: _html/templates/mail.html [hunk 27]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            </td>
+                                                                                                        </tr>
+                                                                                                    </tbody>
+                                                                                                </table>
+                                                                                            </td>
+                                                                                            <td>
+                                                                                                <table align="left" border="0" cellpadding="0" cellspacing="0" style="display:inline;">
+                                                                                                    <tbody><tr>
+                                                                                                        <td valign="top" style="padding-right:10px; padding-bottom:9px;" class="mcnFollowContentItemContainer">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 28]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnFollowContentItem">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 29]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                <tbody><tr>
+                                                                                                                    <td align="left" valign="middle" style="padding-top:5px; padding-right:10px; padding-bottom:5px; padding-left:9px;">
+                                                                                                                        <table align="left" border="0" cellpadding="0" cellspacing="0" width="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Remove width.

File: _html/templates/mail.html [hunk 30]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                            <tbody>
+                                                                                                                                <tr>                                                        
+                                                                                                                                    <td align="center" valign="middle" width="24" class="mcnFollowIconContent">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 31]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                                        <a href="http://www.twitter.com/" target="_blank">
+                                                                                                                                            <img src="http://frontend-news.dev.pitechplus.com/drupal_camp_mail/linkedin.png" style="display:block;" height="24" width="24" class="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Remove class and add alt.

File: _html/templates/mail.html [hunk 32]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            </td>
+                                                                                                        </tr>
+                                                                                                    </tbody>
+                                                                                                </table>
+                                                                                            </td>
+                                                                                            <td class="mobile-exclude">
+                                                                                                <table align="left" border="0" cellpadding="0" cellspacing="0" style="display:inline;">
+                                                                                                    <tbody><tr>
+                                                                                                        <td valign="top" style="padding-right:10px; padding-bottom:9px;" class="mcnFollowContentItemContainer">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 33]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                            <table border="0" cellpadding="0" cellspacing="0" width="100%" class="mcnFollowContentItem">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 34]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                <tbody><tr>
+                                                                                                                    <td align="left" valign="middle" style="padding-top:5px; padding-right:10px; padding-bottom:5px; padding-left:9px;">
+                                                                                                                        <table align="left" border="0" cellpadding="0" cellspacing="0" width="">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Remove empty width.

File: _html/templates/mail.html [hunk 35]
```diff
@@ -0,0 +1,418 @@
...
+                                                                                                                            <tbody>
+                                                                                                                                <tr>                                                        
+                                                                                                                                    <td align="center" valign="middle" width="24" class="mcnFollowIconContent">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Bad class naming . Use hyphen instead of camel case.

File: _html/templates/mail.html [hunk 36]
```diff
@@ -0,0 +1,418 @@
...
+                                                                </tr>
+                                                            </table>
+                                                        </td>
+                                                    </tr>
+                                                </table>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
+                                                <table border="0" cellpadding="0" cellspacing="0" width="100%" class="buttonContainer" style="min-width:100%;">
```
Comment (bajenaru) [inline _html/templates/mail.html]: Please use hyphen naming for class.

File: _html/templates/mail.html [hunk 37]
```diff
@@ -0,0 +1,418 @@
...
+                                                    <tbody>
+                                                        <tr>
+                                                            <td style="padding-top:0; padding-right:18px; padding-bottom:18px; padding-left:18px;" valign="top" align="center">
+                                                                <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate !important;border-radius: 3px;background-color: #FCB02A;">
+                                                                    <tbody>
+                                                                        <tr>
+                                                                            <td align="center" valign="middle" style="font-family: Helvetica; font-size: 18px; padding: 18px;">
+                                                                                <a class="Button" href="http://" target="_self" style="font-weight: bold;letter-spacing: -0.5px;line-height: 100%;text-align: center;text-decoration: none;color: #FFFFFF;">Verify email address</a>
```
Comment (bajenaru) [inline _html/templates/mail.html]: Use lowercase for this class.

**Pull request merged**